docs/lib/storage/fragments/native_common/getting-started/common.md

```diff
@@ -21,14 +21,6 @@ Enter the following when prompted:
 ```console
 ? Please select from one of the below mentioned services:
     `Content (Images, audio, video, etc.)`
-? You need to add auth (Amazon Cognito) to your project in order to add storage for user files. Do you want to add auth now?
```

Comment:
I believe these questions show up when you add `storage` before `auth`.

Reply:
Thanks for catching that. I think I did have `auth` already added before adding `storage`. I'll add those prompts back in.